### ISSUE TITLE
same tiny error code needed to be fixed

### ISSUE THREAD

**Issue by tomsonsgs** [rsethur/DMNPlusTF#1]
def add_bi_dirn_gru_layer(inputs, hidden_size, len_per_input_ph, dropout_in_gru_rnn, dropout_keep_prob_ph):
    gru_cell = tf.nn.rnn_cell.GRUCell(hidden_size)
    gru_cell1 = tf.nn.rnn_cell.GRUCell(hidden_size)
    # apply droput to grus if flag set
    if dropout_in_gru_rnn:
        gru_cell = tf.nn.rnn_cell.DropoutWrapper(gru_cell, input_keep_prob=dropout_keep_prob_ph,
                                                      output_keep_prob=dropout_keep_prob_ph)
        gru_cell1 = tf.nn.rnn_cell.DropoutWrapper(gru_cell1, input_keep_prob=dropout_keep_prob_ph,
                                                      output_keep_prob=dropout_keep_prob_ph)

    outputs, states = tf.nn.bidirectional_dynamic_rnn(gru_cell, gru_cell1, inputs, dtype=np.float32,sequence_length=len_per_input_ph)

    final_state_fw, final_state_bw = states
    outputs_fw, outputs_bw = outputs
    return outputs_fw, outputs_bw, final_state_fw, final_state_bw

with tf.variable_scope("InputFusionLayer"):
            # Dynamic bidirn rnn: facts_from_fusion_layer dim: for fw and bw each: [batch_size, max_sentences_per_input, hidden_size]
            facts_from_fusion_layer_fw, facts_from_fusion_layer_bw, _, _ = add_bi_dirn_gru_layer(inputs, config.hidden_size,
                                                                  sentences_per_input_in_batch_qbt,
                                                                  config.dropout_in_gru_rnn, dropout_keep_prob_placeholder)

            facts_from_fusion_layer = tf.add(facts_from_fusion_layer_fw, facts_from_fusion_layer_bw)

same tiny error code needed to be fixed as showed
